readme.md

```diff
@@ -8,81 +10,94 @@ To lessen the pain in this task, I need a library that will keep track of what t
 
 It stores last-access times in a [LevelUP](https://github.com/rvagg/node-levelup) db - if you don't need persistence, throw [level-mem](https://github.com/Level/level-mem) at it.
 
-Example
-=====
+## breaking changes: version 2.x
+
+- Only tested with LevelUP 2.x
+- Ordered arguments to the constructor are no longer accepted, you gotta pass in an object
+- Emitting events on the expirer object can no longer be done as a proxy for calling the functions
+- All functions return promises
+- Uses Promise + Set + ES2017, polyfill/transpile as needed
+
+# Example
 
 ```js
-    // Some levelUP db
-	var db = level('wat')
-	// Expire stuff after 15 seconds of inactivity
-	var expirer = new Expirer(15000, db)
+// Expire stuff after 15 seconds of inactivity
+const expirer = new Expirer(15 * 1000, someLevelUpDb)
```

Comment:
```js
const expirer = new Expirer({ timeoutMs: 15 * 1000, db: someLevelUpDb })
```

Reply:
oh snap!  That's what I get for not using jsmd.  I didn't want it to be waiting around for the timeouts, though...

Reply:
That's all I see from a very preliminary scan... Might be able to take a better look later.